cli/js/timers.ts

```diff
@@ -18,49 +18,30 @@ interface Timer {
 
```

Comment:
If we land this, there should be a paragraph at the top of this file explaining the algorithm.